docs/en_US/NAS/Cream.md

```diff
@@ -0,0 +1,117 @@
+
+<div align="left">
+  <img src="./../../img/cream.jpg" height="230" alt="Ocean"/><br/>
+  <!-- <p>Example SiamFC, SiamRPN and SiamMask outputs.</p> -->
+</div>
+
+## Reproduction Results
+Top-1 Accuracy on ImageNet. The training with 16 Gpus is a little bit superior than 8 Gpus.
+
+| Model (M Flops) | NNI (8Gpus) | Paper (16Gpus) | 
+| ---- |:-------------:| :-----:|
+| 14M | testing | 59.6 |
+| 42M | 65.8 | 66.5 |
+| 114M | 72.1 | 72.8 |
+| 285M | 76.7 | 77.6 |
+| 470M | 78.9 | 79.2 |
+| 600M | 79.4 | 80.0 |
+
+Top-1 Accuracy on ImageNet reported in the paper, supassing MobileNetV3 and EfficientNet-B0/B1.
+
+<table style="border: none">
+    <th><img src="./../../img/cream_flops100.jpg" alt="drawing" width="400"/></th>
+    <th><img src="./../../img/cream_flops600.jpg" alt="drawing" width="400"/></th>
+</table>
+
+
+## Requirements
+* python >= 3.6
+* torch >= 1.2
+* torchscope
+* apex
+
+## Examples
+
+[Example code](https://github.com/microsoft/nni/tree/master/examples/nas/cream)
+
+Please run the following scripts in the example folder.
+
+## Data Preparation
+
+You need to first download the [ImageNet-2012](http://www.image-net.org/) to the folder `./data/imagenet` and move the validation set to the subfolder `./data/imagenet/val`. To move the validation set, you cloud use the following script: <https://raw.githubusercontent.com/soumith/imagenetloader.torch/master/valprep.sh> 
+
+Put the imagenet data in `./data`. It should be like following:
+
+```
+./data/imagenet/train
+./data/imagenet/val
+...
+```
+
+## Quick Start
+
+### I. Search
+
+First, build environments for searching.
+
+```
+pip install -r ./requirements.txt
+```
+
+To search for an architecture, you need to configure the parameters `flops_minimum` and `flops_maximum` to specify the desired model flops, such as [0,600]MB flops. You can specify the flops interval by changing these two parameters in `./run.sh`
+
+```
+--flops_minimum 0 # Minimum Flops of Architecture
+--flops_maximum 600 # Maximum Flops of Architecture
+```
+
+For example, if you expect to search an architecture with model Flops <= 200M, please set the `flops_minimum` and `flops_maximum` to be `0` and `200`.
+
+After you specify the flops of the architectures you would like to search, you can search an architecture now by running:
+
+```
+sh ./run.sh
+```
+
+Searched model needs to be retrained to obtain the final model. Retraining code will be released soon.

```

Comment:
better to explain how to get the final model, it is a json file or other format?